wikibase/queryService/api/QuerySamples.js

```diff
@@ -6,181 +6,42 @@ wikibase.queryService.api.QuerySamples = ( function ( $ ) {
 	'use strict';
 
 	/**
-	 * QuerySamples API for the Wikibase query service
+	 * QuerySamples API for the Europeana query service
```

Comment:
@SrdjanStevanetic can we have this file under the Europeana folder, and keep the original file in the wikibase folder?